contrib/completion/zsh/_docker-compose

```diff
@@ -198,21 +198,23 @@ __docker-compose_subcommand() {
         (create)
             _arguments \
                 $opts_help \
-                "(--no-recreate --no-build)--force-recreate[Recreate containers even if their configuration and image haven't changed. Incompatible with --no-recreate.]" \
-                "(--force-recreate)--no-build[If containers already exist, don't recreate them. Incompatible with --force-recreate.]" \
-                "(--force-recreate)--no-recreate[Don't build an image, even if it's missing]" \
+                "(--no-recreate)--force-recreate[Recreate containers even if their configuration and image haven't changed. Incompatible with --no-recreate.]" \
```

Comment:
Given that `--force-recreate`, `--no-recreate`, `--no-build` and `--build` are now the same for the `create` and `up` commands, can you extract those options into a separated variable like we do on the docker/docker zsh completion ? (https://github.com/docker/docker/blob/master/contrib/completion/zsh/_docker#L537)